ui/pages/confirmations/confirmation/templates/__snapshots__/error.test.js.snap

```diff
@@ -6,7 +6,7 @@ exports[`error template matches the snapshot 1`] = `
     class="confirmation-page"
   >
     <div
-      class="mm-box confirmation-page__content mm-box--padding-4"
+      class="mm-box confirmation-page__content mm-box--padding-0"
```

Comment:
is this screen related? If so, then I think this and the success test snap update are clear 

![CleanShot 2025-04-15 at 11 57 57](https://github.com/user-attachments/assets/edcf1db8-b01d-4e08-80db-8bdf21c60127)